diff -Nur xnc-4.4.7.orig/Makefile.in xnc-4.4.7.chng/Makefile.in
--- xnc-4.4.7.orig/Makefile.in	Wed Mar 27 13:16:28 2002
+++ xnc-4.4.7.chng/Makefile.in	Mon Nov  4 12:50:08 2002
@@ -256,31 +256,31 @@
 
 install: install-$(PLUGINS)
 	#Installing X Northern Captain...
-	$(INSD) $(INSTDIR)
-	$(INS) ./users.xnc $(INSTDIR)
-	$(INS) ./groups.xnc $(INSTDIR)
-	$(INS) ./rpm.xnc $(INSTDIR)
-	$(INS) ./zips.xnc $(INSTDIR)
-	$(INS) ./gzip.xnc $(INSTDIR)
-	$(INS) ./bzip.xnc $(INSTDIR)
-	$(INS) ./nc $(INSTDIR)/xnc
-	$(INSD) $(INSTMAN)
-	$(INSM) ./yo-man/*.1 $(INSTMAN)/man1
-	$(INS) ./ives/ives $(INSTDIR)
-	$(INS) ./ives/iedit $(INSTDIR)
-	$(INS) ./ives/iview $(INSTDIR)
-	$(INS) ./xsetup/xncsetup $(INSTDIR)
-	$(INS) ./xjroot/xjpegroot $(INSTDIR)
-	$(INS) ./launch/xnlaunch $(INSTDIR)
-	$(INS) ./resloader/xncloader $(INSTDIR)
+	$(INSD) $(DESTDIR)$(INSTDIR)
+	$(INS) ./users.xnc $(DESTDIR)$(INSTDIR)
+	$(INS) ./groups.xnc $(DESTDIR)$(INSTDIR)
+	$(INS) ./rpm.xnc $(DESTDIR)$(INSTDIR)
+	$(INS) ./zips.xnc $(DESTDIR)$(INSTDIR)
+	$(INS) ./gzip.xnc $(DESTDIR)$(INSTDIR)
+	$(INS) ./bzip.xnc $(DESTDIR)$(INSTDIR)
+	$(INS) ./nc $(DESTDIR)$(INSTDIR)/xnc
+	$(INSD) $(DESTDIR)$(INSTMAN)/man1
+	$(INSM) ./yo-man/*.1 $(DESTDIR)$(INSTMAN)/man1
+	$(INS) ./ives/ives $(DESTDIR)$(INSTDIR)
+	$(INS) ./ives/iedit $(DESTDIR)$(INSTDIR)
+	$(INS) ./ives/iview $(DESTDIR)$(INSTDIR)
+	$(INS) ./xsetup/xncsetup $(DESTDIR)$(INSTDIR)
+	$(INS) ./xjroot/xjpegroot $(DESTDIR)$(INSTDIR)
+	$(INS) ./launch/xnlaunch $(DESTDIR)$(INSTDIR)
+	$(INS) ./resloader/xncloader $(DESTDIR)$(INSTDIR)
 	/bin/sh ./replace.script
 	@echo "Don't foget run 'xncsetup' before starting XNC"
 
 
 install-plugins:
-	mkdir -p $(INSTPLUG)
-	$(INS) ./plugins/five/libxncfive.so $(INSTPLUG)
-	$(INS) ./plugins/aqua/libxncaqua.so $(INSTPLUG)
+	mkdir -p $(DESTDIR)$(INSTPLUG)
+	$(INS) ./plugins/five/libxncfive.so $(DESTDIR)$(INSTPLUG)
+	$(INS) ./plugins/aqua/libxncaqua.so $(DESTDIR)$(INSTPLUG)
 
 install-:
 	@echo "Skipping..."
