diff -Nur xnc-4.4.7.orig/configure.in xnc-4.4.7.chng/configure.in
--- xnc-4.4.7.orig/configure.in	Wed Mar 27 16:59:52 2002
+++ xnc-4.4.7.chng/configure.in	Mon Nov  4 12:33:51 2002
@@ -376,7 +376,7 @@
     fprintf(f, "BSD");
     exit(0);
 }], rxvt_cv_ptys=`cat conftestval`, rxvt_cv_ptys=UNKNOWN, dnl
-  AC_MSG_WARN([Define/undefine PTYS_ARE_* in config.h manually]))])
+  AC_MSG_WARN([Def./undef. PTYS_ARE_* in config.h manually]))])
 
 if test x${rxvt_cv_ptys} = xSCO; then
   AC_DEFINE(PTYS_ARE_NUMERIC)
